docs/cross-cutting-concepts/security.md

```diff
@@ -0,0 +1,61 @@
+# Security
+
+This page informs about certain security related topics.
+
+## Disable SBOM Exposing
+
+All services developed with the [templates](/overview#Templates), as well as all ready-to-use components like the [API Gateway](/overview#api-gateway) expose a SBOM (Software Bill of Materials) endpoint at `/actuator/sbom/application` by default.
+
+This can be a problem for closed-source projects using any of the RefArch components, as it exposes information about the software dependencies and might thus lead to expose of critical CVEs.
+
+More information about SBOMs in general and the [CycloneDX](https://cyclonedx.org/) format we use can be found in the [official CycloneDX documentation](https://cyclonedx.org/specification/overview/).
+
+The following sections will describe how to disable the SBOM endpoint in the various components.
+
+### API Gateway
+
+The API Gateway is built using [cyclonedx-maven-plugin](https://github.com/CycloneDX/cyclonedx-maven-plugin) and the generated SBOM file is exposed via the Spring-provided [SBOM actuator endpoint](https://docs.spring.io/spring-boot/api/rest/actuator/sbom.html).
+
+To disable the exposure the endpoint must be disabled through runtime configuration.
+This can be easily achieved by e.g. setting the environment variable `MANAGEMENT_ENDPOINT_SBOM_ACCESS` to `none` to alter the underlying Spring property.
```

Comment:
```suggestion
This can be easily achieved by e.g. setting the environment variable `MANAGEMENT_ENDPOINT_SBOM_ACCESS` to `none` or alter the underlying Spring property.
```